tiledb/tests/test_compat.py

```diff
@@ -46,7 +46,10 @@ def test_compat_tiledb_py_0_5_anon_attr_dense(self):
 
         path = self.path("tiledb_py_0_6_anon_attr")
         with tarfile.open(fileobj=io.BytesIO(base64.b64decode(array_tgz))) as tf:
-            tf.extractall(path)
+            try:
+                tf.extractall(path, filter="fully_trusted")
```

Comment:
Should this order be inverted? ie only use `fully_trusted` if it fails?

Reply:
Hmm, actually looking at the docs I don't think we need both - we control these entirely, so just use `fully_trusted` by default (unless there's some way it could fail...)

Reply:
Oh I see - probably the keyword argument is not valid in those versions. Disregard.